jupyter-on-gke/jupyterhub.tf

```diff
@@ -31,6 +31,15 @@ provider "google-beta" {
   region = var.location
 }
 
+data "google_compute_backend_service" "jupyter-ingress" {
```

Comment:
Could you add comments to each resource we create explaining why we need it/how it's used by IAP?

Reply:
Added some comments